extensions/functions_arithmetic.yaml

```diff
@@ -1563,6 +1563,43 @@ aggregate_functions:
             values: [ TIE_TO_EVEN, TIE_AWAY_FROM_ZERO, TRUNCATE, CEILING, FLOOR ]
         nullability: DECLARED_OUTPUT
         return: fp64?
+  - name: "first"
+    description: >-
+      First value in a set of rows.
+
+      The `null_handling` option determines whether or not null values will be recognized by the function.
+      If `null_handling` is set to `IGNORE_NULLS`, the function will look for the first non-null value.
+      If set to `ACCEPT_NULLS`, the function will return the first value it sees, even if null.
+
+      If all values are null or there are no values, returns null.
+    impls:
+      - args:
+          - name: x
+            value: any
+        options:
+          null_handling:
+            values: [ IGNORE_NULLS, ACCEPT_NULLS ]
+        nullability: DECLARED_OUTPUT
+        decomposable: MANY
+        intermediate: any?
+        return: any?
+  - name: "last"
+    description: >-
+      Last value in a set of rows.
+
+      The `null_handling` option determines whether or not null values will be recognized by the function.
+      If `null_handling` is set to `IGNORE_NULLS`, the function will look for the last non-null value.
+      If set to `ACCEPT_NULLS`, the function will return the last value it sees, even if null.
+
+      If all values are null or there are no values, returns null.
+    impls:
+      - args:
+          - name: x
+            value: any
+        nullability: DECLARED_OUTPUT
+        decomposable: MANY
+        intermediate: any?
+        return: any?
```

Comment:
I think both of these functions should be in `functions_aggregate_generic.yaml` instead of `functions_arithmetic.yaml`